var/spack/repos/builtin/packages/ddt/package.py

```diff
@@ -14,7 +14,7 @@ class Ddt(Package):
     behavior to achieve lightning-fast performance at all scales."""
 
     homepage = "https://arm.com"
-    url = "http://content.allinea.com/downloads/arm-forge-22.0.2-linux-x86_64.tar"
+    url = "https://content.allinea.com/downloads/arm-forge-22.0.2-linux-x86_64.tar"
```

Comment:
I did not succed to `curl -LO` any of URLs (I also try with https://arml.com) but the `spack stage` succeeded: it fetch from `https://mirror.spack.io`

Reply:
@robgics Is there a better link for arm-forge to get ddt, as far as you know?

Reply:
Maybe https://downloads.linaroforge.com/22.1.3/arm-forge-22.1.3-linux-x86_64.tar ?

Reply:
for information: In 2016 Allinea software was acquired by Arm, and in 2023 Linaro acquired Arm Forge.

Reply:
Thansk! Looks similar indeed based on content (though the same version is not available). I'll just do the version addition here, and mark 22.0.2 as deprecated (since still in source mirror).